velox/functions/prestosql/window/LeadLag.cpp

```diff
@@ -128,6 +128,7 @@ class LeadLagFunction : public exec::WindowFunction {
   }
 
   void setRowNumbersForConstantOffset();
+  void setRowNumbersForConstantOffset(int64_t constantOffsetValue);
```

Comment:
- add empty line between methods
- constantOffsetValue -> offset; should the type be vector_size_t?